src/Contract/ConfigProviderInterface.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenEuropa\TaskRunner\Contract;
+
+use Robo\Config\Config;
+
+/**
+ * Interface for configuration providers.
+ *
+ * Classes implementing this interface:
+ * - Should have `TaskRunner\ConfigProviders` as relative namespace. For
+ *   instance when `Some\Namespace` points to the `src/` directory, then the
+ *   class should be placed in `src/TaskRunner/ConfigProviders` and will have
+ *   `Some\Namespace\TaskRunner\ConfigProviders` as namespace.
+ * - The class name should end with the `ConfigProvider` suffix.
+ *
+ * @package OpenEuropa\TaskRunner\Contract
```

Comment:
This `@package` directive in PHPDoc is a relic from the past. It is intended to solve collisions between different classes which have the same name when generating API documentation using PHPDoc. This was something that was sorely needed back in 2002 but is obsolete since namespaces were added to PHP 5.3.0. Ref https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.package.pkg.html

We already use the namespace to provide a logical ordering to the code hierarchy, it is not needed to duplicate this in `@package`. We are also not using PHPDoc in this project.

Reply:
Fixed.